pymeasure/instruments/common_base.py

```diff
@@ -645,23 +645,16 @@ def setting(set_command, docs,
                                   dynamic=dynamic,
                                   )
 
-    def check_errors(self):
-        """ Read all errors from the instrument.
-
-        :return: list of error entries
-        """
-        raise NotImplementedError("Implement it in a subclass.")
```

Comment:
I agree with keeping check_get/set_errors abstract implementations for the interface only. 
I'm not sure I agree with removing `check_errors` completely -- it's the common parent/"interface definition" of both Instrument.check_errors and Channel.check_errors (as much as Python can do such a thing).

Reply:
I added it (and a test) to the common_base.

It's good, that it is there, for another reason as well: Once we remove SCPI's `check_errors` from `Instrument`, the inherited version defines the interface, which instrument impelementations should follow.